### PR TITLE
WEBCORE-121: Fix i18n output in GitHub PR description

### PR DESCRIPTION
**Pull request by kimc0de**
## Issue
https://zattoo2.atlassian.net/browse/WEBCORE-121

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const combineBody = (previousBody, outputText) => {
     if (hasOutput(previousBody)) {
         return previousBody.replace(
             outputRegex,
-            outputText ? `<!-- output start -->\n${outputText}\n<!-- output end -->` : '',
+            outputText ? `\n<!-- output start -->\n${outputText}\n<!-- output end -->` : '',
```
Comment (nitzanashi) [inline src/utils.js:63]: Can also be added to output end

**Review by gotbahn — APPROVED** [review]

File: src/utils.js (hunk 1 above)
Comment (kimc0de) [reply src/utils.js:63]: idk if it's necessary. because it would add uneven space between section. We already have blank line before the i18n title

Comment (nitzanashi) [reply src/utils.js:63]: As you prfer

**Review by nitzanashi — APPROVED** [review]

**Comment by kimc0de** [comment]
@z-merge it

**Comment by nitzanashi** [comment]
@kim you didn't update the `dist` package so it won't effect the action, please create a new pr and update `dist`

**Pull request merged**